app/invocation/invocation_action_card.tsx

```diff
@@ -77,20 +90,15 @@ export default class InvocationActionCardComponent extends React.Component<Props
       action.commandDigest.sizeBytes;
     rpcService
       .fetchBytestreamFile(commandFile, this.props.model.getId(), "arraybuffer")
-      .then((action_buff: any) => {
-        let temp_array = new Uint8Array(action_buff);
+      .then((actionBuff: any) => {
+        let tempArray = new Uint8Array(actionBuff);
         this.setState({
           ...this.state,
-          command: build.bazel.remote.execution.v2.Command.decode(temp_array),
+          command: build.bazel.remote.execution.v2.Command.decode(tempArray),
         });
+        this.fetchInputRoot(action.inputRootDigest);
```

Comment:
Since this only depends on the action and not the command in the response, can this be moved to a separate `fetchInputRoot` function and done in parallel with `fetchCommand`?

Reply:
can i do it after i fetch the command in the fetchAction function?

Reply:
(unresolving)